autogpt_platform/frontend/src/components/monitor/FlowInfo.tsx

```diff
@@ -62,90 +64,30 @@ export const FlowInfo: React.FC<
       (selectedVersion == "all" ? flow.graph_version : selectedVersion),
   );
 
+  const hasInputs = Object.keys(flow.input_schema.properties).length > 0;
+  const hasCredentialsInputs =
+    Object.keys(flow.credentials_input_schema.properties).length > 0;
```

Comment:
Should we be more defensive when accessing `input_schema ` ? Gave us some problems recently 🙇🏽

Reply:
In my opinion: no. The frontend types should match the backend behavior. I have checked this, and they do now. And according to the type definition (based on the backend behavior), `flow.input_schema` is always present and always has a `properties` attribute. Adding `?` without understanding why it's breaking or updating the type is just asking for more bugs to happen.